### PR TITLE
Pun Pun persistence is now configurable

### PR DESCRIPTION
**Pull request by coiax**
- Fixes a bug where a facehugger placed directly onto someone mask slot wouldn't infect them.
- PUNPUN_PERSISTENCE config option added, with options for no persistence, persistence, and persistence BUT NO LIVE FACEHUGGERS KTHX.
  With no config option set, it defaults to persistence.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
probably best as a config :+1:

**Comment by Incoming5643** [comment]
I don't see why the the naming mechanic has to be affected. In any case I don't see this ever getting turned off so whatever

**Comment by Core0verload** [comment]
fffffuuuuuuuuuuuuuuun removal

File: config/game_options.txt [hunk 1]
```diff
@@ -329,6 +329,15 @@ MIDROUND_ANTAG_LIFE_CHECK 0.7
 #NO_SUMMON_MAGIC
 #NO_SUMMON_EVENTS
 
+
+##Pun Pun Persistence
+# This allows Pun Pun, the bartender's monkey, to keep its hat and/or mask from round
+# to round, along with changing its successors name if it died last round.
+# 0 to disable
+# 1 to enable
+# 2 to enable, but turns facehuggers passed into next round into toy facehuggers
```
Comment (duncathan) [inline config/game_options.txt]: Make them defines in the code, please.

**Comment by Shadowlight213** [comment]
@Incoming5643 I might set it to persistence BUT NO LIVE FACEHUGGERS KTHX
I see the bartender often hide punpun in a safe area when he gets a facehugger.
Nuclear ops vs crew vs xeno rounds do not end well.

**Comment by Incoming5643** [comment]
often?

I've seen it happen like... once.

**Comment by coiax** [comment]
This PR is literally because of a request from the King of Basil. It has been happening a lot recently, with this bug that facehugger equipping doesn't infect, along with the recently fixed #17311.

**Comment by KorPhaeron** [comment]
>Nuclear ops vs crew vs xeno rounds do not end well.

That sounds fun though? And also rare. I have only see the pun pun facehugger maybe twice, and once it got put in containment immediatley.

**Comment by KorPhaeron** [comment]
I really do not think this pr is needed

**Comment by coiax** [comment]
@KorPhaeron literally made it because TheLongbowMan pretty much pleaded for it to be made. You should take it up with them.

**Comment by Bawhoppen** [comment]
yes I don't know what its gonna be changed on sybil but I know facehuggers is being turned off on basil

**Comment by KorPhaeron** [comment]
I don't think this is an issue anymore now that the bug is fixed, just a rare silly occurrence.

**Comment by Bawhoppen** [comment]
@KorPhaeron reopen this there was no reason to close it
configs are always good and I know huggers were definitely going to be disabled on basil

**Comment by KorPhaeron** [comment]
KorPhaeron is the pun-pun thing still happening on a regular basis since lamarr got fixed?
TheLongbowMan I don't think so.

**Comment by Bawhoppen** [comment]
yes but it can still happen

**Pull request closed without merging**